README.md

```diff
@@ -1,30 +1,29 @@
 # Optimus
-[![test workflow](https://github.com/odpf/optimus/actions/workflows/test.yml/badge.svg)](test)
```

Comment:
@krishdevdb badges provide useful information about project status, so let's continue to keep them.